snap/build.sh

```diff
@@ -46,4 +46,4 @@ docker run --rm -e "IS_RELEASE_JOB=$IS_RELEASE_JOB" -e "RELEASE=$RELEASE" -e "SN
 
 # also delete the image we created so that it doesn't persist on the system as it contains credentials
 docker image rm --force edgex-snap-builder:latest
-docker image prune
+docker image prune --force
```

Comment:
@anonymouse64 There is a JJB macro you can use in place of this cleanup here https://github.com/edgexfoundry/ci-management/blob/master/jjb/edgex-macros.yaml#L11-L20
You would use it like this https://github.com/edgexfoundry/ci-management/blob/master/jjb/edgex-templates-go.yaml#L202, that is, call it directly after the build script.

Reply:
Yeah I like that solution better, I think I will close this for now and do this in the ci-management repo and re-open this with the image handling stuff removed from `snap/build.sh`